File: QViz.NET/Periscope.QViz/JSON/APIBodyValue.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test API Body Value Details
	/// </summary>
	public class APIBodyValue
	{
		/// <summary>
		/// Identifier of the API Body Value Object
		/// </summary>
		public string bodyValueId;

		/// <summary>
		/// API Body Value as JSON String
		/// </summary>
		public string jsonString;

		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;

		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APIBodyValue()
		{
			this.bodyValueId = "";
			this.jsonString = "";
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APIBodyValue.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for Test API Body Value Details
 */
public class APIBodyValue {

	/**
	 * Identifier of the API Body Value Object
	 */
	public String bodyValueId;

	/**
	 * API Body Value as JSON String
	 */
	public String jsonString;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * Default Constructor
	 */
	public APIBodyValue() {
		this.bodyValueId = "";
		this.jsonString = "";
		this.createdOn = null;
		this.updatedOn = null;
	}

}
